Assignment1/Activity1.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="utf-8"?>​
+<myMessage>​
+  <myMessage>
```

Comment:
1. don't use same name for root and nested elements - bad practice
2. myMessage does not make sense here. you store info about people, not messages